options.c

```diff
@@ -1938,6 +1953,7 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 		.lname	= "IO offset",
 		.alias	= "fileoffset",
 		.type	= FIO_OPT_STR_VAL,
+		.cb	= str_offset_cb,
```

Comment:
Thanks for the comment.  I believe this is addressed by reducing the starting_offset_percent to a `uint8_t` in a subsequent commit.